servicetalk-http-utils/src/main/java/io/servicetalk/http/utils/BeforeFinallyHttpOperator.java

```diff
@@ -37,7 +37,7 @@
  * is dictated by the terminal event of the payload body. If the body is subscribed to multiple times, only the first
  * subscribe will receive ownership of the terminal events.</p>
  *
- * Example usage tracking the begin and end of a request:
+ * Example usage tracking the beginning and ending of a request:
```

Comment:
Consider applying similar changes in `AfterFinallyHttpOperator` javadoc

Reply:
Good eye: I'd intended to do it but it slipped my mind.